### ISSUE TITLE
Support StartAfter(List<dynamic> values) 

### ISSUE THREAD

**Issue by IwaoAkiba** [atn832/cloud_firestore_mocks#41]
Seems not support startAfter(List<dynamic> values).

  ```
int d1 = 1;
  int d2 = 2;
  int d3 = 3;

  var collectionRef =
      Firestore.instance.collection("users/");

  await collectionRef.add({'created': d1});
  await collectionRef.add({'created': d2});
  await collectionRef.add({'created': d3});

  var results =
      await collectionRef.orderBy('created', descending: true).startAfter([d2]).getDocuments();

  results.documents.forEach((d) => print(d.data['created']));
```

**Comment by atn832** [comment]
Thanks for writing a unit test!

You're right, this feature is not supported so far. I might get to it in the future. If you're feeling adventurous, feel free to contribute a PR!

**Comment by atn832** [comment]
@Hadii1 graciously implemented this feature at #70.
